### ISSUE TITLE
Add support for `aria-labeledby` (not labeLLedby)

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4284]
This synonym attribute seems to have full support in the three major browser engines now:
https://github.com/w3c/aria/issues/2093

Should do some screen reader testing, but from the sound of if this should no longer fail axe-core's button-name rule:

```html
<span hidden id="search">Search</span>
<button class="search-icon" aria-labeledby="search"></button>
```

**Comment by cookiecrook** [comment]
Or fail with a warning ("Attribute misspelled") instead of an error.

**Comment by cookiecrook** [comment]
Actually looking at the [results in Gecko (click show details)](https://wpt.fyi/results/accname/name/comp_labelledby.html?label=pr_head&max-count=1&pr=43698), Firefox's implementation of the non-standard spelling is very incomplete, so it's possible the ARIA WG decision may not be to document and keep it. Probably best for Axe Core not to change until that decision is made.
